### ISSUE TITLE
Omitting state interpolation when persist isn't set

### ISSUE THREAD

**Issue by jakubrohleder** [tonyhb/redux-ui#30]
First of all - good job guys! I really like idea of this package!

I've found some disturbing bug (feature?): when the persist option isn't set sometimes state variables with functional value are not interpolated. Here is the example:

``` javascript
import React, { Component, PropTypes } from 'react';
import ui from 'redux-ui';

@ui({
  key: 'some-name',
  // persist: true,
  state: {
    uiVar1: () => 1
  }
}) export default class TestUIComponent extends Component {
  static propTypes = {
    ui: PropTypes.object.isRequired,
  };

  render() {
    const { ui: { uiVar1 } } = this.props;
    console.log(uiVar1);

    return (
      <div />
    );
  }
}
```

The console log look as follows:

```
TestUIComponent.js?34cd:98: 1
TestUIComponent.js?34cd:98: 1
TestUIComponent.js?34cd:98: 1
TestUIComponent.js?34cd:98: uiVar1() {
      return 1;
    }
TestUIComponent.js?34cd:98: uiVar1() {
      return 1;
    }
```

In case you can't reproduce this bug, I'm using this boilerplate: [react-redux-universal-hot-example](https://github.com/erikras/react-redux-universal-hot-example).

**Comment by tonyhb** [comment]
Thanks for the heads up, going to take a look at this early next week. Running a release at Docker and things are heating up over the next few days!

**Comment by tonyhb** [comment]
@jakubrohleder This should also be fixed by #36 — let me know if it persists after 0.0.15!

**Comment by tonyhb** [comment]
@jakubrohleder This is actually covered by #37 — we'll release a new version soon!
